ballerina-tests/http-security-tests/tests/ssl_sni_host_name_test.bal

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2024 WSO2 Inc. (http://www.wso2.org).
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/test;
+import ballerina/http_test_common as common;
+
+listener http:Listener http2SniListener = new (9207, http2SslServiceConf);
```

Comment:
Move the ports to this file as constants: https://github.com/ballerina-platform/module-ballerina-http/blob/master/ballerina-tests/http-security-tests/tests/test_service_ports.bal